lib/container.ts

```diff
@@ -311,7 +311,6 @@ export class Container extends (EventEmitter as {
 					// check the string itself. Any path ending with / is quite clearly
 					// a directory, and it means that we can avoid making another docker
 					// call
-					// TODO: Think about how this is affected when using windows
```

Comment:
I think this TODO no longer required, as `copy.dest` should now always be POSIX compliant.

Reply:
Yeah, let's remove this